pkgs/os-specific/linux/evdi/default.nix

```diff
@@ -1,37 +1,40 @@
-{ stdenv, fetchFromGitHub, kernel, libdrm }:
+{ stdenv, fetchFromGitHub, fetchgit, fetchpatch, kernel, libdrm }:
 
 stdenv.mkDerivation rec {
   pname = "evdi";
-  version = "1.6.4";
+  version = "-git-2019-01-16";
 
   src = fetchFromGitHub {
     owner = "DisplayLink";
     repo = pname;
-    rev = "v${version}";
-    sha256 = "1yrjm8lvvz3v4h5af6m9qzq6z4lbgd7qbvq5rz7sjhdsh7g6qibd";
+    rev = "391f1f71e4c86fc18de27947c78e02b5e3e9f128";
+    sha256 = "147cwmk57ldchvzr06lila6av7jvcdggs9jgifqscklp9x6dc4ny";
   };
 
   nativeBuildInputs = kernel.moduleBuildDependencies;
 
   buildInputs = [ kernel libdrm ];
 
-  makeFlags = [
-    "KDIR=${kernel.dev}/lib/modules/${kernel.modDirVersion}/build"
-    "KVER=${kernel.modDirVersion}"
+  patchFlags = [ "-Np1" ];
+  patches = [
+     (fetchpatch {   url    = "https://crazy.dev.frugalware.org/evdi-all-in-one-fixes.patch";
+                     sha256 = "03hs68v8c2akf8a4rc02m15fzyp14ay70rcx8kwg2y98qkqh7w30"; })
   ];
 
+  makeFlags = [ "KVER=${kernel.modDirVersion}" "KDIR=${kernel.dev}/lib/modules/${kernel.modDirVersion}/build" ];
+
   hardeningDisable = [ "format" "pic" "fortify" ];
```

Comment:
Are you sure these are still needed? Would be great if we could one (or all) of them

Reply:
Not sure what you mean by "these" but then I have no idea... I just modified current to get it working with git and the patch.